sw/rust/ibex-demo-system-pac/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+#![no_std]
+#![allow(non_camel_case_types)]
+
+pub use ibex_demo_system_pac::generic::*;
+pub use ibex_demo_system_pac::*;
+
+include!(concat!(env!("OUT_DIR"), "/lib.rs"));
```

Comment:
Need new line at end of file -- this should be fixed automatically by `cargo fmt`

Reply:
Done